sklearn/externals/_lobpcg.py

```diff
@@ -22,11 +22,12 @@
 
```

Comment:
I will revert those changes where `black` decided that I should not only change a single line ;)